src/ApplicationManager.cpp

```
@@ -537,8 +537,11 @@ void ApplicationManager::handleSpoolUpdated(const AppMessage& msg) {
     if (display_) {
         if (msg.payload.spoolUpdated.success) {
             char line1[17];
-            snprintf(line1, sizeof(line1), "Updated: %.0fg",
-                     kgRemaining * 1000.0f);
+            if (kgRemaining > 0.0f) {
```

Clang (14.0.6)

[note] +3, including nesting penalty of 2, nesting level increased to 3

(clang)